CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changes in Fbx Exporter
 
+## [5.1.0] - 2023-09-08
```

Comment:
Maybe we should update this to a closer date to the actual expected release?

Reply:
I can change to today's date which is 2023-09-18 together with another small CI change.